py-polars/tests/unit/test_datelike.py

```diff
@@ -539,6 +539,14 @@ def test_date_range() -> None:
     assert result.cast(pl.Utf8)[-1] == "2022-01-01 00:00:59.247379260"
 
 
+def test_range_invalid_unit() -> None:
+    msg = "unit: 'D' not supported. Available units are: 'ns', 'us', 'ms', 's', 'm', 'h', 'd', 'w', 'mo', 'y', 'i'"
```

Comment:
FYI: I probably wouldn't match quite so exactly on the full error string.
`"'D' not supported"` would be sufficient 👍

Reply:
😄 sure thing!